### ISSUE TITLE
cheat engine wont set up

### ISSUE THREAD

**Issue by eldritchFyou** [cheat-engine/cheat-engine#1292]
iv redownloaded it. run as admin. everything I can think of and it shuts off in a few seconds after iv opened it

**Comment by cheat-engine** [comment]
uninstall badlion and restart windows

**Comment by eldritchFyou** [comment]
badlion?

**Comment by eldritchFyou** [comment]
ok I've done that, it still freezes up and closes like its crashing

**Comment by cheat-engine** [comment]
Make sure you uninstall using the badlion uninstaller. Just deleting files won't fix the other parts badlion infects.

Also, restart windows afterwards. Do not shut down and back on, actual restart. Windows 10 fakes shutdown.  If you can't then try unplugging the powercord while windows is on

**Comment by eldritchFyou** [comment]
ok... what the hell is badlion

**Comment by cheat-engine** [comment]
badlion is a minecraft client that installs a device driver on your system that watches what you do (and who knows what else) and will decide what you can and can not run. Cheat Engine is in the "can not run" list

**Comment by eldritchFyou** [comment]
yeah, I don't have that. I use fabric loader ill try uninstalling that

**Comment by cheat-engine** [comment]
do a 'driverquery /v > bla.txt' and then upload bla.txt here. It may show what else it could be

**Comment by eldritchFyou** [comment]
I never thought making dark souls 3 fun again would be so hard

**Comment by eldritchFyou** [comment]
I'm in drivers but I can't find anything about loads

**Comment by eldritchFyou** [comment]
im also going through command prompt and cant find anything called bala.text

**Comment by eldritchFyou** [comment]
maybe if i try an older version it will work

**Comment by eldritchFyou** [comment]
it did. 6.38 installed perfectly

**Comment by NextGenerationHackers** [comment]
I am also having the same issue with CE 7.1. I don't have badlion since I have never installed or even played Minecraft. As soon as I click on the .exe and after I give it permission in the UAC (User Account Control), a icon in Taskbar shows up, but within Seconds... It disappears. Following the PID in Task Manager, it starts the installation and then it stops.

**Comment by cheat-engine** [comment]
Might be a firewall issue that is blocking the installer to connect to the server

**Comment by NextGenerationHackers** [comment]
Could you please debug this. I would like to use CE 7.1. Thank You.

**Comment by ScotianOne** [comment]
what im seeing is, if anything is blocking "dpd.securestudies.com" the installation closes.. for instance, a pihole with default settings would filter this, firewalls, or even just a failure to connect for any other reason. In my case I simply whitelisted the domain on my dns server and it worked perfectly afterward.
